sbncode/BeamSpillInfoRetriever/job/bnbspillinfo.fcl

```diff
@@ -8,7 +8,7 @@ bnbspillinfo: {
     URL: "" #keep this blank and we're good
     Bundle: "BoosterNeutrinoBeam_read"
     MultiWireBundle: "BNBMultiWire"
-    TimeWindow: "500" #seconds
+    TimeWindow: "700" #seconds
```

Comment:
This was updated because with the trigger database we were able to actually realize we were not grabbing all the spills, that additional 200 seconds has allowed us to make sure we are grabbing all the correct spills between the two majority triggers